en/setup/dev_env_linux.md

```diff
@@ -126,7 +199,7 @@ ${HEXAGON_SDK_ROOT}/tools/debug/mini-dm/Linux_Debug/mini-dm
 
 Note: Alternatively, especially on Mac, you can also use [nano-dm](https://github.com/kevinmehall/nano-dm).
 
-### Raspberry Pi hardware
+## Raspberry Pi Hardware
 
 Developers working on Raspberry Pi hardware need to download a ARMv7 cross-compiler, either GCC or clang.
```

Comment:
Note, as above, clang-3.5 lldb-3.5 are no longer installed in simulator dependencies. SHould they be here?